### PR TITLE
Performance

### PR DESCRIPTION
**Pull request by ajacksified**
:eyeglasses: @danehansen

### CODE REVIEW

File: assets/js/client.es6.js [hunk 1]
```diff
@@ -25,11 +25,23 @@ import getTimes from '../../src/lib/timing';
...
+      initialize(false);
+    }
+
+    head.appendChild(shimScript, document.currentScript);
+
+    shimScript.src = global.bootstrap.assetPath + '/js/es5-shims.js';
+  }, 1);
 } else {
-  initialize(true);
+  window.addEventListener('load', function() {
```
Comment (danehansen) [inline assets/js/client.es6.js]: is this able to fire sooner than on window load? in theory that wouldn't happen until all images have loaded, but maybe that doesn't apply in our react app...

**Comment by ajacksified** [comment]
CSS: were 2 requests totalling 28.1k, now is 21.3 in one request
JS: about ~60k smaller after gzip (20%.)

**Comment by ajacksified** [comment]
:haircut:

**Comment by ajacksified** [comment]
CSS: were 2 requests totalling 28.1k, now is 21.3 in one request
JS: about ~60k smaller after gzip (20% reduction), 90k before gzip
HTML: about ~60k instead of ~80k.

**Comment by ajacksified** [comment]
That last commit depends on https://github.com/reddit/horse-react/pull/3.

Needs another pass to clean up some of the SVG rendering and HTML escaping, but is almost there.

**Comment by ajacksified** [comment]
:eyeglasses: @curioussavage

**Pull request merged**